cmd/podman/images/pull.go

```diff
@@ -93,23 +93,22 @@ func pullFlags(flags *pflag.FlagSet) {
 
 // imagePull is implement the command for pulling images.
 func imagePull(cmd *cobra.Command, args []string) error {
-	pullOptsAPI := pullOptions.ImagePullOptions
 	// TLS verification in c/image is controlled via a `types.OptionalBool`
 	// which allows for distinguishing among set-true, set-false, unspecified
 	// which is important to implement a sane way of dealing with defaults of
 	// boolean CLI flags.
 	if cmd.Flags().Changed("tls-verify") {
-		pullOptsAPI.TLSVerify = types.NewOptionalBool(pullOptions.TLSVerifyCLI)
+		pullOptions.SkipTLSVerify = types.NewOptionalBool(!pullOptions.TLSVerifyCLI)
```

Comment:
Maybe I've just gotten used to it over the years, but I'd rather not negate this, personal bias me thinks and ¯\_(ツ)_/¯

Reply:
I completely share your feeling. The trouble here is that we have to negate it at some point. c/image has a skip logic while the CLI has a "do" logic. As we regressed in the backend, I moved the negation here (CLI only) in hope to never touch it again. The backend will likely more often be touched, so I think it's good to have a type/logic that follows that c/image expects rather than negating there.